### PR TITLE
fix stem.mod on chords

### PR DESCRIPTION
**Pull request by rettinghaus**
Haven't found this case in real life, but should work anyway.

### CODE REVIEW

File: src/view_element.cpp [hunk 1]
```diff
@@ -1482,6 +1482,15 @@ void View::DrawStem(DeviceContext *dc, LayerElement *element, Layer *layer, Staf
         if (stem->GetStemMod() == STEMMODIFIER_sprech) {
             int yShift = m_doc->GetDrawingDoubleUnit(staff->m_drawingStaffSize) * 2;
             Note *note = vrv_cast<Note *>(stem->GetParent());
+            if (!note) {
+                Chord *chord = vrv_cast<Chord *>(stem->GetParent());
```
Comment (lpugin) [inline src/view_element.cpp:1486]: Add assert for chord. Thanks

**Comment by lpugin** [comment]
Even less likely: a chord with `@stem.mod="spreech"` and no note. I think it will crash with your code. Pls double check this.

File: src/view_element.cpp (hunk 1 above)
Comment (rettinghaus) [reply src/view_element.cpp:1486]: done

**Comment by rettinghaus** [comment]
Verovio automatically adds a note to empty chords, so everything works fine now:
Sprechakkord

```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="https://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="https://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="4.0.0">
   <meiHead>
      <fileDesc>
         <titleStmt>
            <title>Sprechakkorde</title>
         </titleStmt>
         <pubStmt>
         </pubStmt>
      </fileDesc>
   </meiHead>
   <music>
      <body>
         <mdiv>
            <score>
               <scoreDef>
                  <staffGrp>
                     <staffGrp>
                        <staffDef n="1" lines="5" clef.shape="G" clef.line="2" key.sig="0" meter.count="4" meter.unit="4" />
                     </staffGrp>
                  </staffGrp>
               </scoreDef>
               <section>
                  <measure n="1">
                     <staff n="1">
                        <layer n="1">
                           <chord dur="4" stem.mod="sprech">
                              <note />
                              <note />
                           </chord>
                           <chord dur="4" stem.mod="sprech">
                              <note />
                           </chord>
                           <chord dur="4" stem.mod="sprech">
                           </chord>
                           <chord dur="4" stem.len="0" stem.mod="sprech">
                              <note />
                              <note />
                           </chord>
                           <chord dur="4" stem.visible="false" stem.mod="sprech">
                              <note />
                              <note />
                           </chord>
                        </layer>
                     </staff>
                  </measure>
               </section>
            </score>
         </mdiv>
      </body>
   </music>
</mei>
```

**Pull request merged**